build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -3,6 +3,10 @@ kind: ClusterServiceVersion
 metadata:
   name: __NAME__.v__VERSION__
   namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
```

Comment:
Do we support this in Maistra as well? Should this be limited to the ServiceMesh CSV?

Reply:
I'll create an issue to investigate, we have a couple of weeks left